domain-models-interface-extensions/src/main/java/org/folio/rest/tools/ClientGenerator.java

```diff
@@ -260,10 +265,10 @@ public void generateClassMeta(String className) {
       okapiUrl = jc.field(JMod.PRIVATE, String.class, OKAPI_URL);
 
       /* class variable to http options */
-      JFieldVar options = jc.field(JMod.PRIVATE, HttpClientOptions.class, "options");
+      JFieldVar options = jc.field(JMod.PRIVATE, WebClientOptions.class, "options");
 
       /* class variable to http client */
-      JFieldVar httpClient = jc.field(JMod.PRIVATE, HttpClient.class, "httpClient");
+      JFieldVar httpClient = jc.field(JMod.PRIVATE, WebClient.class, "httpClient");
```

Comment:
```suggestion
      JFieldVar webClient = jc.field(JMod.PRIVATE, WebClient.class, "webClient");
```
All `httpClient` usages should be renamed to `webClient` to reflect reality.